amd_gpu/gpu.c

```diff
@@ -163,10 +163,12 @@ const char* err_to_str(cl_int ret)
 		return "CL_INVALID_LINKER_OPTIONS";
 	case CL_INVALID_DEVICE_PARTITION_COUNT:
 		return "CL_INVALID_DEVICE_PARTITION_COUNT";
+#ifndef __APPLE__
```

Comment:
I fixed it in #15 because this is an issue with OpenCl < 2.0 and independent of the APPSDK.